posthog/models/file_system/unfiled_file_saver.py

```diff
@@ -0,0 +1,87 @@
+# posthog/models/file_system/unfiled_file_saver.py
+
+from typing import Optional
+
+from posthog.models.team import Team
+from posthog.models.user import User
+from posthog.models.file_system.file_system import FileSystem, split_path, escape_path
+from posthog.models.file_system.file_system_mixin import FileSystemSyncMixin
+
+from posthog.models.feature_flag import FeatureFlag
+from posthog.models.experiment import Experiment
+from posthog.models.insight import Insight
+from posthog.models.dashboard import Dashboard
+from posthog.models.notebook import Notebook
+
+MIXIN_MODELS = {
+    "feature_flag": FeatureFlag,
+    "experiment": Experiment,
+    "insight": Insight,
+    "dashboard": Dashboard,
+    "notebook": Notebook,
```

Comment:
can haz recording playlist? :)

Reply:
Absolutely! I kept the currently implemented for this PR, and the next PRs will add a lot more